packages/examples/cvat/exchange-oracle/src/chain/kvstore.py

```diff
@@ -21,3 +23,24 @@ def get_job_launcher_url(chain_id: int, escrow_address: str) -> str:
     escrow = get_escrow(chain_id, escrow_address)
 
     return OperatorUtils.get_leader(ChainId(chain_id), escrow.launcher).webhook_url
+
+
+def set_public_key_to_kvstore() -> None:
+    if Config.encryption_config.pgp_public_key_url:
+        for network_config in Config.get_network_configs(only_configured=True):
+            w3 = get_web3(network_config.chain_id)
+            kvstore_client = KVStoreClient(w3)
+
+            is_pub_key_stored = False
+            try:
+                is_pub_key_stored = kvstore_client.get_public_key(network_config.addr)
```

Comment:
Should the returned value be compared with what would be written? Why not just write it every time?

Reply:
> Should the returned value be compared with what would be written?

Probably yes, I'll fix

> Why not just write it every time?

Because write operation to KVstore is not free